packages/wrangler/src/__tests__/pages/deploy.test.ts

```diff
@@ -20,7 +20,7 @@ import { normalizeProgressSteps } from "./project-upload.test";
 import type { Project, UploadPayloadFile } from "../../pages/types";
 import type { RestRequest } from "msw";
 
-describe("deployment create", () => {
+describe("pages deploy", () => {
```

Comment:
God help us all, but we could really use some spring cleaning in this test suite 🍠